extensions/amp-story/1.0/amp-story-interactive.js

```diff
@@ -688,16 +705,32 @@ export class AmpStoryInteractive extends AMP.BaseElement {
   }
 
   /**
-   * Updates the selected classes on option selected.
-   * @param {!Element} selectedOption
-   * @private
+   * Updates the selected classes on component and option selected.
+   * @param {?Element} selectedOption
+   * @protected
    */
   updateToPostSelectionState_(selectedOption) {
     this.rootEl_.classList.add('i-amphtml-story-interactive-post-selection');
-    selectedOption.classList.add('i-amphtml-story-interactive-option-selected');
+    selectedOption
+      ? selectedOption.classList.add(
+          'i-amphtml-story-interactive-option-selected'
+        )
+      : null;
```

Comment:
If called with null, it will set to post-select state without selecting an option. Used in the results page